govcd/external_network_v2_test.go

```diff
@@ -0,0 +1,170 @@
+// +build extnetwork network functional openapi ALL
+
+/*
+ * Copyright 2020 VMware, Inc.  All rights reserved.  Licensed under the Apache v2 License.
+ */
+
+package govcd
+
+import (
+	"fmt"
+
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+	. "gopkg.in/check.v1"
+)
+
+func (vcd *TestVCD) Test_CreateExternalNetworkV2Nsxt(check *C) {
+	vcd.testCreateExternalNetworkV2Nsxt(check, vcd.config.VCD.Nsxt.Tier0router, types.ExternalNetworkBackingTypeNsxtTier0Router)
+}
+
+func (vcd *TestVCD) Test_CreateExternalNetworkV2NsxtVrf(check *C) {
+	// Checking NSX-T VRF router would be expected to require
```

Comment:
Could you rephrase? - I didn't grasp idea of this message

Reply:
Changed.